src/es-router.js

```diff
@@ -73,7 +73,9 @@ class EsRouter {
     }
 
     //do an initial routing
-    this.path(this.getPathFromUrl());
+    if (routeOnLoad !== false) {
```

Comment:
ah, yeah, though then on `routeOnLoad` you should do`, routeOnLoad = true}`, that would make more logical sense